docs/BR.md

```diff
@@ -3147,44 +3180,96 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
+|     `issuer`               | MUST     | MUST be byte-for-byte identical to the `subject` field of the Issuing CA. |
+|     `thisUpdate`           | MUST     | Indicates the issue date of the CRL. |
+|     `nextUpdate`           | MUST     | Indicates the date by which the next CRL will be issued. For CRLs covering Subscriber Certificates, at most 10 days after the `thisUpdate`. For other CRLs, at most 12 months after the `thisUpdate`. |
+|     `revokedCertificates`  | *        | MUST be present if the CA has issued a certificate that has been revoked and the corresponding entry has yet to appear on at least one regularly scheduled CRL beyond the revoked certificate's validity period. The CA SHOULD remove an entry for a corresponding certificate after it has appeared on at least one regularly scheduled CRL beyond the revoked certificate's validity period. See the "revokedCertificates Component" table for additional requirements.  |
+|     `extensions`           | MUST     | See the "CRL Extensions" table for additional requirements. |
+| `signatureAlgorithm`       | MUST     | Encoded value MUST be byte-for-byte identical to the `tbsCertList.signature`. |
+| `signature`                | MUST     | - |
+| Any other value            | NOT RECOMMENDED | - |
+
 ### 7.2.1 Version number(s)
 
+Certificate Revocation Lists MUST be of type X.509 v2.
+
 ### 7.2.2 CRL and CRL entry extensions
 
-1. `reasonCode` (OID 2.5.29.21)
+Table: CRL Extensions
+
+| __Extension__                     | __Presence__    | __Critical__          | __Description__ |
+| ----                              | -               | -                     | ----- |
+| `authorityKeyIdentifier`          | MUST            | N                     | See [Section 7.1.2.11.1](#712111-authority-key-identifier) |
+| `CRLNumber`                       | MUST            | N                     | MUST contain an INTEGER greater than or equal to zero (0) and less than 2¹⁵⁹, and convey a strictly increasing sequence.        |
+| `IssuingDistributionPoint`        | *           | Y                     | See [Section 7.2.2.1 CRL Issuing Distribution Point](#7221-crl-issuing-distribution-point) |
+| Any other extension               | NOT RECOMMENDED        | -                     |       |
+
+Table: revokedCertificates Component
 
-   If present, this extension MUST NOT be marked critical.
+| __Component__                     | __Presence__    | __Description__ |
+| ----                              | -               | ----- |
+| `serialNumber`                    | MUST            | MUST be byte-for-byte identical to the serialNumber contained in the revoked certificate. |
+| `revocationDate`                  | MUST            | Normally, the date and time revocation occurred. See the footnote following this table for circumstances where backdating is permitted.  | 
+| `crlEntryExtensions`              | *               | See the "crlEntryExtensions Component" table for additional requirements. |
```

Comment:
nit: column width
```suggestion
| __Component__        | __Presence__ | __Description__ |
| ----                              | -               | ----- |
| `serialNumber`       | MUST         | MUST be byte-for-byte identical to the serialNumber contained in the revoked certificate. |
| `revocationDate`     | MUST         | Normally, the date and time revocation occurred. See the footnote following this table for circumstances where backdating is permitted.  | 
| `crlEntryExtensions` | *            | See the "crlEntryExtensions Component" table for additional requirements. |
```

Reply:
Addressed in https://github.com/ryancdickson/staging/commit/bbf5155dd87de476eb360a79feb8e49fdd0eefe6.